client/payment-methods/style.scss

```diff
@@ -27,4 +27,27 @@
 			margin-right: 1px; // 1px to accommodate for box-shadow
 		}
 	}
+
+	&__express-checkouts {
```

Comment:
Do you mean to add padding on the right side of the text so that the text wraps to another line in normal view. If it's in mobile view, then we push the background image out from view port, leaving just the purple background?

Reply:
Updated so that the text no longer hover on top of the background image:
![image](https://user-images.githubusercontent.com/572862/124981552-ef039080-dff2-11eb-926d-bc66b7404689.png)

![image](https://user-images.githubusercontent.com/572862/124981540-ec08a000-dff2-11eb-8c06-e22acdacfc32.png)